api.py

```
@@ -0,0 +1,108 @@
+import asyncio
+import logging
+import os
```

Ruff (0.8.2)

`os` imported but unused

Remove unused import: `os`

(F401)